fem/post/create_res_sim.py

```diff
@@ -199,18 +220,26 @@ def extract_image_plane(snic, axes, ele_pos):
       ele_pos: elevation position for plane of interest
```

Comment:
@AnnaEKnight docstring inputs need to be updated for new keyword arguments and details about what the valid input values mean